docs/metrics/object-detection.md

```diff
@@ -0,0 +1,121 @@
+
+Evaluation of object detection models requires ground truths and model inferences. The ground truths in an image are
+objects outlined by bounding boxes each labelled with a class, typically annotated by human labellers. The model
+inferences for an image are the labelled bounding boxes having confidence scores (the model's certainty of
+correctness), as if the model annotated the image.
+
+A [bounding box matcher](./geometry-matching.md) can align an image's ground truths and model inferences to produce
+[TP / FP / FN counts](./tp-fp-fn-tn.md). These counts are foundational for computing more detailed metrics, which
+provide insight into the model performance. Different evaluation configurations can be tuned to filter out model
+inferences before/after employing a matching algorithm such as filtering out inferences with a confidence score under
+0.01 and/or ignoring matches where the [IoU](./iou.md) (the overlap between the ground truth and and inference
+bounding box) under 0.5.
+
+Below are some of the commonly used metrics for evaluating object detection models:
+
+1. [**Precision**](./precision.md): Precision measures the ratio of correctly detected objects to all objects detected
+    by the model. High precision indicates a low rate of false positives.
+
+    $$\text{Precision} = \frac{\text{TP}}{\text{TP} + \text{FP}}$$
+
+2. [**Recall**](./recall.md): Recall measures the ratio of correctly detected objects to all actual objects (the
+    annotations). High recall indicates that the model is good at detecting most of the objects labelled by humans.
+
+    $$\text{Recall} = \frac{\text{TP}}{\text{TP} + \text{FN}}$$
+
+3. [**F1-Score**](./f1-score.md): F1-Score is the harmonic mean of precision and recall - a balance of both metrics as
+    one metric.
+
+    $$
+    \begin{align}
+    \text{F}_1 &= \frac {2} {\frac {1} {\text{Precision}} + \frac {1} {\text{Recall}}} \\[1em]
+    &= \frac {2 \times \text{Precision} \times \text{Recall}} {\text{Precision} + \text{Recall}}
+    \end{align}
+    $$
+
+4. [**Mean Average Precision (mAP)**](./averaging-methods.md): Mean average precision (mAP) is obtained by first
+    computing the [average precision (AP)](./average-precision.md) for each class based on
+    [Precision-Recall (PR) curves](./pr-curve.md) and then macro-averaging those scores across all classes. mAP is a
+    comprehensive indicator of a model's performance across multiple categories.
+
+    $$
+    \begin{align}
+    \text{mAP} = \frac{1}{N} \sum_{\text{class}}^{\text{all classes}} \text{AP}_\text{class}
+    \end{align}
+    $$
+
+    !!! info "Guide: Averaging Methods"
+
+        Read the [averaging methods](./averaging-methods.md) guide if you're not familiar with "macro" and "micro"
+        terminology.
+
+## Evaluation Plots for Object Detection Models
+
+There are several common plots used to analyze the performance of object detection models.
+
+??? info "Advanced Plot Insights"
```

Comment:
Should this be under "1. Precision Recall (PR) Curves"?

Reply:
This is meant to be a generic recommendation to any plot. Good flag - probably lives best under `For example: an F1-Score vs confidence threshold plot, a histogram of IoUs, or mAP across different IoUs.`